arangod/MMFiles/MMFilesHashIndex.cpp

```diff
@@ -468,6 +468,14 @@ Result MMFilesHashIndex::insert(transaction::Methods* trx, LocalDocumentId const
 /// @brief removes an entry from the hash array part of the hash index
 Result MMFilesHashIndex::remove(transaction::Methods* trx, LocalDocumentId const& documentId,
                                 VPackSlice const& doc, OperationMode mode) {
+#ifdef ARANGODB_ENABLE_FAILURE_TESTS
```

Comment:
You can safely remove the `#ifdef` here, because the macro `TRI_IF_FAILURE` contains that ifdef itself.